migration/files/03AddDocumentIndex.go

```diff
@@ -0,0 +1,79 @@
+package migrationfiles
+
+import (
+	"strings"
+
+	"github.com/centrifuge/go-centrifuge/documents"
```

Comment:
Can we avoid importing go-centrifuge types here? The initial idea of this migration tool is to be as standalone as possible, without depending on external data structures. Can you create an intermediate model with only what this migration needs and then store the structure using the low level db functions?

Reply:
This is required for us to update the model safely with the index. Getting a model out requires us to register the types else the fetch will fail. IMO, as long as we don't have any specific DB imports, I think we can safely do this without any issues.

Reply:
I understand the need to register the models if we are going to use them :) . I would like to have this library lower level, since depending how it goes, it might be a candidate to opensource itself.

Reply:
The imports are part of the migration files. There will always be specific implemetations in the migration functions. If we plan to open source this, we can open source the wrapper and utils and expect users to inject the migration files.

Reply:
You are right, I thought it was in the parent package. All good then!